frontend/src/ProductsOffers/ProductOfferEntry/ProductOfferEntry.tsx

```diff
@@ -0,0 +1,70 @@
+import { Button, Container, Grid, Stack, Typography } from "@mui/material"
+import ProductOfferEntryDate, { IProductOffersEntryDate } from "./ProductOfferEntryDate"
+import ProductOffersLocation, { IProductOffersLocation } from "./ProductOfferEntryLocation"
+import ProductOffersPrice, { IProductOffersPrice } from "./ProductOfferEntryPrice"
+import ProductOffersUser, { IProductOffersUser } from "./ProductOfferEntryUser"
+
+const mainContainerStyles = { backgroundColor: "#cee8ab", borderRadius: 20 }
+
+export interface IProductOffersEntry {
+    id: string
+    addDate: IProductOffersEntryDate,
+    expirationDate: IProductOffersEntryDate,
+    user: IProductOffersUser,
+    imgSrc: string,
+    productName: string,
+    location: IProductOffersLocation,
+    price: IProductOffersPrice
+}
```

Comment:
i have a problem with that. This object is really big and contains a lot of diffrent information. On the other hand on this moment i don't see a way to do it better. Leave this comment for some disscusion